### PR TITLE
[KO] Translation of rendering-on-the-web.md

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
This translation has passed a high-level review by Alconost
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/ko/review/8472)

### CODE REVIEW

**Comment by agektmr** [comment]
Sorry for keeping you wait for a long time, but we finally got your translation reviewed and ready to merge to Web Fundamentals. Can you please sign the CLA? https://cla.developers.google.com/clas
@marocchino @riseshia @secondjay

Thanks.
cc: @petele

File: src/content/ko/updates/2019/02/rendering-on-the-web.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+[서비스 작업자](https://developers.google.com/web/fundamentals/primers/service-workers/)가 선택할 수 있는 옵션인 경우 “삼자형” 렌더링에 관심을 가질 수 있습니다. 초기/비 JS 메뉴를 위해 스트리밍 서버 렌더링을 사용할 수 있으며, 서비스 작업자는 설치를 완료한 후 메뉴를 위한 HTML 렌더링을 수행합니다. 이를 통해 캐시 구성요소와 템플릿의 최신 업데이트 상태를 유지하며, 동일한 세션에서 신규 보기를 렌더링하기 위해 SPA 스타일의 메뉴를 활성화합니다. 이와 같은 접근 방법은 사용자가 서버, 클라인트 페이지 및 서비스 작업자 간의 동일한 템플릿 및 라우팅 코드를 공유할 때 효과가 높습니다.
+
+<img src="../../images/2019/02/rendering-on-the-web/trisomorphic.png" alt="Diagram of Trisomorphic rendering, showing a browser and service worker
+communicating with the server">
+
+## 검색 엔진 최적화 고려 {: #seo}
+
+웹 렌더링 전략을 선택할 때 보통 SEO 영향을 고려합니다. 크롤러가 쉽게 해석할 수있는 "완성된" 경험을 제공하기 위해 서버 렌더링이 종종 선택됩니다. 크롤러는 [JavaScript를 해석할 수도](https://web.dev/discoverable/how-search-works) 있지만 여기에는 알아둘만한 [제한 사항](/search/docs/guides/rendering)이 있습니다. 그것은 클라이언트 측 렌더링은 작동하지만 추가 테스트 및 여타 작업(leg work)이 없을 수 있다는 것 입니다. 최근에 [동적 렌더링](/search/docs/guides/dynamic-rendering)은 아키텍처가 클라이언트 측 JavaScript에 크게 의존하는 경우 고려할 가치가 있는 옵션이 되었습니다.
+
+[모바일 친화적인 테스트](https://search.google.com/test/mobile-friendly) 도구는 선택한 방식이 원하는대로 작동하는지 테스트하는 데 사용할 수 있습니다. Google 크롤러에 표시되는 페이지, 자바 스크립트 실행 후 발견 된 일련의 HTML 콘텐츠 및 렌더링 중 발생한 모든 오류를 시각적으로 미리 보여줍니다.
```
Comment (marocchino) [inline src/content/ko/updates/2019/02/rendering-on-the-web.md]: This test is to determine how mobile friendly your application is. However, "모바일 친화적인 테스트" can be mistaken for the test's own affinity.
```suggestion
[모바일 친화성 테스트](https://search.google.com/test/mobile-friendly) 도구는 선택한 방식이 원하는대로 작동하는지 테스트하는 데 사용할 수 있습니다. Google 크롤러에 표시되는 페이지, 자바 스크립트 실행 후 발견 된 일련의 HTML 콘텐츠 및 렌더링 중 발생한 모든 오류를 시각적으로 미리 보여줍니다.
```

File: src/content/ko/updates/2019/02/rendering-on-the-web.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+### 부분 Rehydration {: #partial-rehydration }
+
+부분 Rehydration은 수행 과정이 어렵습니다. 개별 요소(구성요소/보기/트리)는 점진적으로 Rehydration 및 분석되고, 상호 활동이 거의 없거나 반응성을 확인할 수 없는 경우 이 방법을 통해 점진적인 Rehydration의 개념을 확장합니다. 대부분의 정적인 부분의 경우 일치하는 JavaScript 코드는 비활성 참조 및 장식 기능성으로 전환되어 클라이언트의 자취가 거의 없는 상태가 됩니다. 부분 Hydration 방법은 자체적인 문제와 손상을 포함합니다. 캐시에 대한 흥미로운 문제가 발생하며, 클라이언트 측의 메뉴는 애플리케이션의 비활성 부분을 위한 서버 렌더링 방식의 HTML을 추측할 수 없습니다. 서버 렌더링 방식의 HTML은 애플리케이션의 비활성 부분을 위해서 전체 페이지를 로드하지 않고 접근할 수 있습니다.
+
+### 삼자형 렌더링 {: #trisomorphic }
+
+[서비스 작업자](https://developers.google.com/web/fundamentals/primers/service-workers/)가 선택할 수 있는 옵션인 경우 “삼자형” 렌더링에 관심을 가질 수 있습니다. 초기/비 JS 메뉴를 위해 스트리밍 서버 렌더링을 사용할 수 있으며, 서비스 작업자는 설치를 완료한 후 메뉴를 위한 HTML 렌더링을 수행합니다. 이를 통해 캐시 구성요소와 템플릿의 최신 업데이트 상태를 유지하며, 동일한 세션에서 신규 보기를 렌더링하기 위해 SPA 스타일의 메뉴를 활성화합니다. 이와 같은 접근 방법은 사용자가 서버, 클라인트 페이지 및 서비스 작업자 간의 동일한 템플릿 및 라우팅 코드를 공유할 때 효과가 높습니다.
```
Comment (marocchino) [inline src/content/ko/updates/2019/02/rendering-on-the-web.md]:
```suggestion
[서비스 작업자](https://developers.google.com/web/fundamentals/primers/service-workers/)가 선택할 수 있는 옵션인 경우 “삼자형” 렌더링에 관심을 가질 수 있습니다. 초기의 JS로 만들어지지 않은 메뉴에 스트리밍 서버 렌더링을 사용할 수 있으며, 서비스 작업자는 설치를 완료한 후 메뉴를 위한 HTML 렌더링을 수행합니다. 이를 통해 캐시 구성요소와 템플릿의 최신 업데이트 상태를 유지하며, 동일한 세션에서 신규 보기를 렌더링하기 위해 SPA 스타일의 메뉴를 활성화합니다. 이와 같은 접근 방법은 사용자가 서버, 클라인트 페이지 및 서비스 작업자 간의 동일한 템플릿 및 라우팅 코드를 공유할 때 효과가 높습니다.
```

Comment (marocchino) [reply src/content/ko/updates/2019/02/rendering-on-the-web.md]: Question to Korean speaker: "서비스 작업자" feel natural to you?

**Comment by marocchino** [comment]
@agektmr Finally, thank you for your interest. ❤️  I am anxious because I have no immediate reaction. but, I applied CLA.

File: src/content/ko/updates/2019/02/rendering-on-the-web.md (hunk 2 above)
Comment (riseshia) [reply src/content/ko/updates/2019/02/rendering-on-the-web.md]: IMO, it's not. "서비스 워커" is major at this point, and I don't feel to try to re-wording with perfect translated korean word ;)

Comment (agektmr) [reply src/content/ko/updates/2019/02/rendering-on-the-web.md]: @marocchino can you fix this?

File: src/content/ko/updates/2019/02/rendering-on-the-web.md (hunk 1 above)
Comment (agektmr) [reply src/content/ko/updates/2019/02/rendering-on-the-web.md]: @marocchino is there anything left to do here?

**Comment by agektmr** [comment]
@marocchino as soon as you address issues listed above, I can merge your translation. Sorry it took so long!

**Comment by ilyaspiridonov** [comment]
@agektmr As this has been stale for a while, but has passed a high-level review by Alconost - can we merge this one? Contributors are welcome to improve translations via GitLocalize later.

**Comment by marocchino** [comment]
Oh, I missed this one. 😅 
This PR is not open to me and I have no permission to modify.
My review didn't fix anything, but I think it's good to merge and fix it later.

**Pull request merged**